package.json

```diff
@@ -41,10 +41,14 @@
     "discord-api-types": "^0.12.1",
     "eslint": "^7.15.0",
     "eslint-config-marine": "^7.2.0",
+    "eslint-plugin-tsdoc": "^0.2.11",
     "typescript": "^4.2.0-dev.20210204"
   },
   "eslintConfig": {
     "extends": "marine",
+    "plugins": [
+      "eslint-plugin-tsdoc"
```

Comment:
I'm not entirely sure this is set up correctly? If I try to revert line 34/35 in `src/DataStore.ts` back to `@param voiceChannel ...` then no linter errors appear.